exegol/model/ContainerConfig.py

```diff
@@ -354,6 +382,20 @@ def disableExegolResources(self):
             self.__exegol_resources = False
             self.removeVolume(container_path='/opt/resources')
 
+    def enableShellLogging(self):
+        """Procedure to enable exegol shell logging feature"""
+        if not self.__shell_logging:
+            logger.verbose("Config: Enabling shell logging")
+            self.__shell_logging = True
+            self.addLabel(self.__label_features.get('enableShellLogging'), "Enabled")
+
+    def __disableShellLogging(self):
+        """Procedure to enable exegol shell logging feature"""
```

Comment:
Small typo here, enable -> disable
Also, it seems the enable and disable functions have different naming format, was it on purpose?

Reply:
got it :+1:

Reply:
Type fixed in #[81963dc](https://github.com/ShutdownRepo/Exegol/pull/115/commits/81963dc8f1acabbb8b749e680c505fa045d8a8b1)